bonxai_core/src/pointer_input_stream.hpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Contributors to the Bonxai Project
+ * Copyright Contributors to the OpenVDB Project
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <streambuf>
+
+namespace Bonxai {
+struct PointerStreamBuffer final : public std::streambuf {
```

Comment:
Add some doumentation. i don't know what the purpose of this is

Reply:
Fixed